src/main/java/picard/sam/util/SamComparison.java

```diff
@@ -155,12 +168,18 @@ private boolean compareAlignmentsAndCatalogDuplicateMarkingDifferences() {
         if (!compareValues(leftReader.getFileHeader().getSortOrder(), rightReader.getFileHeader().getSortOrder(),
                 "Sort Order")) {
             System.out.println("Cannot compare alignments if sort orders differ.");
+            if (samComparisonArgumentCollection.OUTPUT_MQ_CONCORDANCE != null) {
```

Comment:
this doesn't need a separate message.  Stating that the alignments cannot be compared (above) covers that the mapping qualities of the alignments aren't being compared.